org-cyf-guides/content/employability/cvs/_index.md

```diff
@@ -4,3 +4,7 @@ title: CVs
 description: Learn how to tailor your CV to the right employer
 weight: 2
 ---
+
+The purpose of a CV is to show an employer that you would be a good person to work with.
+
+Your CV should tell a story about who you are and why you are a good person to work with.
```

Comment:
Should we add a phrase of: "A recruiter will skim through your CV for around 10 seconds. So you must stand out with your CV."